chapters/packaging.Rmd

```diff
@@ -57,7 +57,7 @@ A package consists of one or more Python source files\index{Python package!struc
 in a specific directory structure
 combined with installation instructions for the computer.
 Python packages can come from various sources:
-some are distributed with Python itself,
+some are distributed with Python itself as part of the [Standard Library][python-standard-library],
```

Comment:
how about "as part of the language's [standard library][python-standard-library]" ?